cli/running/lua/launcher.lua

```diff
@@ -12,12 +12,63 @@
 -- Several useful links:
 -- https://porkmail.org/era/unix/arg-max.html
 -- https://unix.stackexchange.com/a/120842
-
+local fun   = require('fun')
 local os = require('os')
 local console = require('console')
 local log = require('log')
 local title = require('title')
 
+local function reducer(res, l, r)
+    if res ~= nil then
+        return res
+    end
+    if tonumber(l) == tonumber(r) then
+        return nil
+    end
+    return tonumber(l) > tonumber(r)
+end
+
+
+local function split(self, sep)
+    local sep, fields = sep or ":", {}
+    local pattern = string.format("([^%s]+)", sep)
+    self:gsub(pattern, function(c) table.insert(fields, c) end)
+    return fields
+end
+
+local function split_version(version_string)
+    local vtable  = split(version_string, '.')
+    local vtable2 = split(vtable[3],  '-')
+    vtable[3], vtable[4] = vtable2[1], vtable2[2]
+    return vtable
+end
+
+--- Returns if version of tarantool is greater then expected.
+local function check_version(expected, version)
+    local iter = fun.iter
+    version = version or _TARANTOOL
+    if type(version) == 'string' then
+        version = split_version(version)
+    end
+    local res = iter(version):zip(expected):reduce(reducer, nil)
+    if res or res == nil then res = true end
+    return res
+end
+
+local origin_cfg = box.cfg
+
+--- Wrapper for cfg to push our values over default.
+local function cfg_wrapper(cfg)
+    local data_cfg = {}
+    data_cfg.wal_dir = os.getenv('TT_WAL_DIR')
+    data_cfg.memtx_dir = os.getenv('TT_MEMTX_DIR')
+    data_cfg.vinyl_dir = os.getenv('TT_VINYL_DIR')
+    -- Forcing new cfg.
+    local success, data = pcall(origin_cfg, data_cfg)
+    if not success then 
+        log.error("wrapping failed")
```

Comment:
I don't think a message like that would be helpful to the user. Start the sentence with a capital letter.